hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/scm/client/ScmClient.java

```diff
@@ -305,6 +305,24 @@ Pipeline createReplicationPipeline(HddsProtos.ReplicationType type,
    */
   boolean getReplicationManagerStatus() throws IOException;
 
+  /**
+   * Start ContainerBalancer.
+   */
+  void startContainerBalancer(double threshold, int idleiterations,
```

Comment:
Can we use K-V map to hold all parameters here, to avoid API change in future in case introducting new parameters